src/probability/stopping.lean

```diff
@@ -739,6 +739,18 @@ lemma le_measurable_space_of_const_le (hτ : is_stopping_time f τ) {i : ι} (h
 
 end preorder
 
+instance sigma_finite_stopping_time {ι} [semilattice_sup ι] [order_bot ι]
```

Comment:
This instance can actually apply, and it allowed me to remove several `sigma_finite` assumptions in #14065 . I added a temporary lemma to that other branch a minute ago to illustrate this (see the last commit): the new lemma `condexp_stopping_time_restrict_eq_of_order_bot` is almost the same as `condexp_stopping_time_restrict_eq` but thanks to the additional `order_bot` hypothesis, the instance triggers and I don't need to supply  `sigma_finite (μ.trim hτ.measurable_space_le)`.

I think that `hτ` can be found since it is present in the goal `sigma_finite (μ.trim hτ.measurable_space_le)`.

Reply:
You're right, thanks for the clarification!